tests/BUILD.bazel

```diff
@@ -77,6 +77,32 @@ sh_test(
     ],
 )
 
+genrule(
+    name = "generate_test",
+    outs = ["generated_file.txt"],
+    cmd = "echo this is a test file > $@",
+)
+
+installer(
+    name = "generated_file_installer_under_test",
+    data = ["generated_file.txt"],
```

Comment:
```
data = [":generated_file"],
```

https://docs.bazel.build/versions/main/skylark/build-style.html#references-to-targets-in-the-current-package

Same below (L97, L101)